packages/tasit-sdk/package.json

```diff
@@ -18,9 +18,12 @@
   },
   "scripts": {
     "prepare": "npx babel src --out-dir dist --ignore **/*.test.js",
-    "test": "npm run test:local",
-    "test:local": "npm run lint && npx mocha src/*.test.js --exclude src/*forkedRopsten.test.js --require @babel/register --file src/testHelpers/mochaSetup.js --no-timeouts",
-    "test:forkedRopsten": "npm run lint && npx mocha src/*forkedRopsten.test.js --require @babel/register --file src/testHelpers/mochaSetup.js --no-timeouts",
+    "test": "npm run lint && npm run test:local",
+    "test:local": "cp src/config/development.js src/config/default.js && npm run test:run",
+    "test:forked-testnet": "npm run test:forked-ropsten",
+    "test:forked-ropsten": "cp src/config/forked-ropsten.js src/config/default.js && npm run test:run",
+    "test:forked-goerli": "cp src/config/forked-goerli.js src/config/default.js && npm run test:run",
+    "test:run": "npx mocha src/*.test.js  --require @babel/register --file src/testHelpers/mochaSetup.js --no-timeouts",
```

Comment:
The naming of this is confusing compared with just `test`

Reply:
Since we have three blockchains to test (empty, forked ropsten and forked goerli) I've made that way to avoid duplication. Here some change suggestions, what do you prefer?

1)
```json
"test": "npm run lint && npm run test:empty-blockchain",
"test:empty-blockchain": "cp src/config/development.js src/config/default.js && npm run test:run",
"test:forked-ropsten": "cp src/config/forked-ropsten.js src/config/default.js && npm run test:run",
"test:forked-goerli": "cp src/config/forked-goerli.js src/config/default.js && npm run test:run",
"test:run": "npx mocha src/*.test.js  --require @babel/register --file src/testHelpers/mochaSetup.js --no-timeouts",
```

2)
```json
"test": "cp src/config/development.js src/config/default.js && npm run test:run",
"test:forked-ropsten": "cp src/config/forked-ropsten.js src/config/default.js && npm run test:run",
"test:forked-goerli": "cp src/config/forked-goerli.js src/config/default.js && npm run test:run",
"test:run": "npx mocha src/*.test.js  --require @babel/register --file src/testHelpers/mochaSetup.js --no-timeouts",
```

3)
```json
"test": "cp src/config/development.js src/config/default.js && npx mocha src/*.test.js  --require @babel/register --file src/testHelpers/mochaSetup.js --no-timeouts",
"test:forked-ropsten": "cp src/config/forked-ropsten.js src/config/default.js && npx mocha src/*.test.js  --require @babel/register --file src/testHelpers/mochaSetup.js --no-timeouts",
"test:forked-goerli": "cp src/config/forked-goerli.js src/config/default.js && npx mocha src/*.test.js  --require @babel/register --file src/testHelpers/mochaSetup.js --no-timeouts",

```

Reply:
Feel free to keep `test:forked-testnet` too as a convenient script.

Ah ok, I see the purpose of `test:run` now as a means to DRY things up. I like option 1. Does that work for you?

I suppose some of the config-copying could move over to a pretest script rather than test, but with a similar approach?

Reply:
And how about that?

```json
    "setup:local": "cp src/config/development.js src/config/default.js",
    "setup:forked-ropsten": "cp src/config/forked-ropsten.js src/config/default.js",
    "setup:forked-goerli": "cp src/config/forked-goerli.js src/config/default.js",
    "test": "npm run setup:local && npm run mocha",
    "test:forked-testnet": "npm run setup:forked-ropsten && npm run mocha",
    "mocha": "npx mocha src/*.test.js  --require @babel/register --file src/testHelpers/mochaSetup.js --no-timeouts",
```

Reply:
@marcelomorgado I like that!

Reply:
Feel free to resolve this

Reply:
Resolving this.